4/john-wiegley-captions.srt

```diff
@@ -783,7 +791,7 @@ Could not match.
 
 179
 00:12:30,800 --> 00:12:36,200
-Oh! because inaudible in io.
+Oh! because [inaudible] in IO.

```

Comment:
I still cannot make it out (they are talking at the same time and I think it would help if I knew more about Haskell).

I think what he means is that the type returned by `parseTest` is `IO`,
so that is why `evalExpr (parseTest expr "1 + 2")` does not parse, since `evalExpr` expects an `Expr`.